core/src/main/java/org/apache/brooklyn/core/mgmt/entitlement/WebEntitlementContext.java

```diff
@@ -38,16 +48,25 @@
     final String requestUniqueIdentifier;
     
     public WebEntitlementContext(String user, String sourceIp, String requestUri, String requestUniqueIdentifier) {
+        this(user, sourceIp, requestUri, requestUniqueIdentifier, new HashMap<>());
```

Comment:
There's an inconsistency with the initialisation of attributes - here it's a hashmap and later it's a mutablemap

Reply:
I'll use HashMap in `WebEntitlementContext`